### DISCUSSION TITLE
restore between aws regions

### DISCUSSION THREAD

**Discussion by geeth4sg** [vmware-tanzu/velero#4492] [Community support Q&A]
i am newbie to velero. I want to restore the cluster  backup from s3  - eu-london to us-east region.

**Comment by Satsank** [comment]
May want to review this thread... make sure the backup-location for both clusters are pointing to the same S3 repository. 

https://github.com/vmware-tanzu/velero/discussions/4501

Same idea here if you are restoring from one cluster in one region to another region. In general, you are better off choosing a backup storage that has no egress costs.
